AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -707,6 +711,13 @@ NS_ASSUME_NONNULL_END
 @property (atomic, assign)           BOOL accessibilityViewIsModal;
 @property (atomic, assign)           BOOL shouldGroupAccessibilityChildren;
 
+@property (nonatomic) UIAccessibilityNavigationStyle accessibilityNavigationStyle;
```

Comment:
Properties above should probably be changed to be nonatomic.  One of the strings is strong instead of copy, too.  For alignment, maybe we should put , nullable at the end.